ccaas_builder/cmd/build/main.go

```diff
@@ -0,0 +1,175 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+	"text/template"
+
+	"github.com/otiai10/copy"
+)
+
+var logger = log.New(os.Stderr, "", 0)
+
+type ChaincodeMetadata struct {
+	Type string `json:"type"`
+}
+
+// Connection structure is used to represent the
+// connection.json file that is supplied in the
+// chaincode package.
+type Connection struct {
+	Address     string `json:"address"`
+	DialTimeout string `json:"dial_timeout"`
+	TLS         bool   `json:"tls_required"`
+	ClientAuth  bool   `json:"client_auth_required"`
+	RootCert    string `json:"root_cert"`
+	ClientKey   string `json:"client_key"`
+	ClientCert  string `json:"client_cert"`
+}
+
+type Config struct {
+	PeerName string
+}
+
+func main() {
+	logger.Println("::Build")
+
+	if err := run(); err != nil {
+		logger.Printf("::Error: %v\n", err)
+		os.Exit(1)
+	} else {
+		logger.Printf("::Type detected as ccaas")
```

Comment:
This looks like a copy/paste from detect program.
And no need for an else here.
See guidance at https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow